docs/docs/intro.md

```diff
@@ -8,11 +8,12 @@ A client library for Cardano in Java. This library simplifies the interaction wi
 
 :::info
 
-**Latest Stable Version**: [0.6.3](https://github.com/bloxbean/cardano-client-lib/releases/tag/v0.6.3)
+**Latest Stable Version**: [0.7.0](https://github.com/bloxbean/cardano-client-lib/releases/tag/v0.7.0)
 
 :::
 
 ### Recent Posts
+- [Cardano Indexing Made Simple: Yaci Store Plugin System for Non-Java Developers](https://satran004.medium.com/cardano-indexing-made-simple-yaci-store-plugin-system-for-non-java-developers-39d4c148ffb1)
```

Comment:
@ducpm2303 Let's remove this. This should be part of Yaci Store documentation. Otherwise, the PR looks good to me. Thanks.